library/src/main/java/com/novoda/downloadmanager/DownloadBatchStatusNotificationDispatcher.java

```diff
@@ -21,7 +21,7 @@
         this.downloadBatchIdNotificationSeen = downloadBatchIdNotificationSeen;
     }
 
-    void updateNotification(DownloadBatchStatus downloadBatchStatus) {
+    void updateNotification(InternalDownloadBatchStatus downloadBatchStatus) {
```

Comment:
This class is not available to clients so passing `InternalDownloadBatchStatus` is ok here.